main.go

```diff
@@ -110,10 +112,20 @@ func main() {
 		}
 	}()
 
+	proxy := validation.New()
```

Comment:
`proxy := validation.New()` this is a very confusing line of code
It's a proxy but it instantiates a new `validation`?